File: CsProjects/CS_Works/PrimeNumbers/PrimeNumber.cs
using System;

namespace PrimeNumbers
{
    class PrimeNumber
    {
        static void Main(string[] args)
        {
            int mod = 0, counter = 0;

            Console.WriteLine("Enter Number:");
            int num = Convert.ToInt32(Console.ReadLine());

            for (int i = 1; i < num; i++)
            {
                mod = num % i;
                if (mod == 0)
                {
                    counter++;
                }
            }
            if (counter > 1)
            {
                Console.WriteLine("Not A Prime Number");
            }
            else
            {
                Console.WriteLine("Prime Number");
            }
        }
    }
}

File: PycharmProjects/PythonBasics/PrimeNumber.py
print("Enter A Number:")
num = int(input())
i, counter = 1, 0
while i != num:
    mod = num % i
    if mod == 0:
        counter = counter + 1
    i = i + 1
if counter == 1:
    print("Prime Number")
else:
    print("Not Prime")